sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/README.md

```diff
@@ -81,6 +81,9 @@ For the event processor client to make use of Azure Storage blobs for checkpoint
 - [Mocking Client Types](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample08_MockingClientTypes.md)  
   A demonstration of how to mock the types in the Event Hubs Processor client library, focusing on common application scenarios.
 
+- [Using the `EventProcessorClient` with an ASP.NET hosted service](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/aspnethostedservice/README.md)  
```

Comment:
This URL will likely continue to cause analysis to fail because it is correctly an explicit reference to `main`  but the path doesn't yet exist.   Once we get to the point where it's our only remaining failure, we can decide whether to force bypass of the failure or comment it out and bring it back after this merges.